docs/cluster_spec.md

```diff
@@ -34,6 +34,34 @@ spec:
       idleTimeoutSeconds: 300
 ```
 
+### etcd v3 & tls
+
+Although kops doesn't presently default to etcdv3 both v3 and TLS can be enabled via the cluster spec (manifests only i.e. no command line options as yet). An upfront warning; at present no upgrade path exists for migrating from v2 to v3 so **DO NOT** try to enable this on a v2 running cluster. The below example snippet assumes a HA cluster of three masters.
```

Comment:
Perhaps: Although kops doesn't presently default to etcd3, it is possible to turn on both v3 and TLS authentication for communication amongst cluster members. These options may be enabled via the cluster spec...